final-assignment-tests/shell/test-037.sh

```diff
@@ -0,0 +1,17 @@
+#/bin/bash
+
+dir=long-path-123456
+count=128
+
+function test_input {
```

Comment:
I think there should be a comment about the resulting path not exceeding `PATH_MAX` (on whatever system this is run).

Reply:
I looked into limits.h and the PATH_MAX is different for different platforms, so I took the smallest one. Not sure how to make this portable.

Reply:
That is cautious approach. Also, different file systems have different limits. These warrant some comment.